<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.999990000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QListWidget" name="chooseNonStandardAcid">
        <item>
         <property name="text">
          <string>3,3,5-triiodothyronine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3,4-dihydroxyproline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3,5-dichlorothyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3,5-diiodotyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-bromo-5-chlorotyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-bromothyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-chlorothyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-hydroxyproline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-iodotyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-methylhistidine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4-hydroxyproline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NG,N'G-dimethylarginine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NG,NG-dimethylarginine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NG-methylargynine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>citrulline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>delta-hydroxylizine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>desmosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dithyrosine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>epsilon-(N,N)-dimethyllysine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>epsilon-(N,N,N)-trimethyl-delta-hydroxylisine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>epsilon-(N,N,N)-trimethyllysine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>epsilon-N-methyllysine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gamma-carboxyglutamic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gipuzine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>isodesmosin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lisinoroleucine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>thyroxine</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioAddToTheEnd">
            <property name="text">
             <string>Add to the end</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAddToTheFront">
            <property name="text">
             <string>Add to the front</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonDeleteEnd">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete from the end</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDeleteFront">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete from the front</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="chooseAminoAcid">
        <item>
         <property name="text">
          <string>A     Alanine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C     Cysteine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D     Aspartic acid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E     Glutamic acid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F     Phenylalanine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G    Glycine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H    Histidine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I      Isoleucine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>K     Lysine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L     Leucine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M    Methionine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N    Asparagine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P    Proline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q    Glutamine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R    Arginine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S    Serine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T    Threonine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>V    Valine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>W   Tryptophan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y    Tyrosine</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="addProtein">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Add new protein</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rmProtein">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Remove last protein</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="SaveProtein">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save protein</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoadProtein">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Load and add protein</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="sumSequence">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Join protein sequence</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
